saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -285,28 +282,23 @@ def _refresh_lines_unit_price(cls, info, instance, cleaned_input, new_instance):
         if new_instance:
             # It is a new instance, all new lines have already updated prices.
             return
+
         shipping_address = cleaned_input.get("shipping_address")
```

Comment:
I think that there is a little bit messy here. We drop calculation of prices but we still have methods like `_refresh_lines_unit_price`, which can be a little bit confusing as we don't refresh anything, only invalidating a flag that determines if prices will need a new fetch.